src/CMakeLists.txt

```diff
@@ -33,6 +33,14 @@ endif ()
 if (REST)
     pkg_check_modules(SOUP REQUIRED libsoup-2.4)
     pkg_check_modules(QRENCODE REQUIRED libqrencode)
+    set(REST_PACKAGES
```

Comment:
CMake was always is mystery for me, but why aren't you adding `libqrencode` here as well?

Reply:
Because the [libqrencode VAPI](https://github.com/pdfpc/pdfpc/blob/31c452c70f5a2133bd1d273f5a723304cb796bdf/src/libqrencode.vapi) is part of pdfpc. Frankly, I was also confused at first :).